### ISSUE TITLE
INDEX NAME TOO LONG FOR ORACLE 19 - KEYCLOAK 24.0.0

### ISSUE THREAD

**Issue by maquineh** [keycloak/keycloak#29580]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

import-export

### Describe the bug

We are trying migrate keycloak 19.0 to 24.0 but when we execute migration in normal way, we got a database error.
We got the error because a index name for oracle 19 index name length have more than 30 characters in jpa-changelog.xml. We can change the name in jpa-changelog.xml file but we gonna lose all the database configuration compatibility.

The index name is FED_USER_ATTR_LONG_VALUES_LOWER_CASE for the table FED_USER_ATTRIBUTE

### Version

24.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

Migration without errors of index name length

### Actual behavior

We are getting a database error ORA-00972: identifier is too long

FED_USER_ATTR_LONG_VALUES_LOWER_CASE (Oracle 19: 30; Liquibase: 36)

### How to Reproduce?

we are just running keycloak 24 with our prior keycloak 19 schema. When we migrate keycloak 11 to 15 and 15 to 19 we didnt get any errors.

### Anything else?

Same problem probably exists also for index `USER_ATTR_LONG_VALUES_LOWER_CASE` which is also more than 30 characters long.

**Comment by mposolda** [comment]
Looks like a regression of https://github.com/keycloak/keycloak/commit/76453550 . It seems same issue happens also for the index `USER_ATTR_LONG_VALUES_LOWER_CASE` , which is also more than 30 characters long.

**Comment by vramik** [comment]
@mposolda @maquineh this bug report seems to be a duplicate of #27590 (and #28371). 

TL;DR Since Oracle 12.2 there is support for identifiers longer than 30. Don't you have set a compatibility mode?
